src/dmd/backend/optabgen.d

```diff
@@ -170,6 +170,7 @@ void dotytab()
 
     f = fopen("tytab.d","w");
 
+/+
```

Comment:
I would recommend you delete this code, but since the whole file is likely to disappear soon, I guess it doesn't matter.

Reply:
The build won't work if this file isn't generated, even if it's empty.